exchangesync/node_policy.go

```diff
@@ -168,7 +175,14 @@ func SetDefaultNodePolicy(config *config.HorizonConfig, pDevice *persistence.Exc
 	if err != nil {
 		glog.V(2).Infof("Failed to retrieve node policy from local db: %v", err)
 	}
-	builtinNodePol, builtinNodePolReadWrite := externalpolicy.CreateNodeBuiltInPolicy(false, false, existingPol)
+	isCluster := false
+	exchDevice, err := persistence.FindExchangeDevice(db)
```

Comment:
and again, I think you can eliminate the need to read the device obj and just call pDevice.IsEdgeCluster() where you need it.

Reply:
fixed